packages/nimble-components/src/toggle-tip/specs/toggle-tip-ixd.md

```diff
@@ -0,0 +1,141 @@
+    -   If the image is defined using `<svg>` element, use `role="img"` and `aria-label` attribute to provide a role and an accessible name for the element.
+    -   Note: Providing ARIA based role and attribute on SVG image ensures robust support across different environments.
+-   The button **MUST** be defined with aria-pressed attribute to convey the state.
+    -   In pressed state – the value of aria-pressed attribute **MUST** be set to “true”.
+    -   When not in pressed state – the value of aria-pressed attribute **MUST** be set to “false”.
+
+#### Popover
+
+-   The container, in which the toggletip content is defined, **SHOULD** be specified with ARIA Live region using `role="status"`. Alternatively, `aria-live="polite"` along with `aria-atomic="true"` **CAN** be specified to the container.
+-   When the toggletip content appears, the container **MUST NOT** prevent the user from accessing content surrounding the toggletip.
+-   Toggletip **SHOULD** show/hide information on pressing the space or enter key.
+-   A `tabindex="0"` **CAN** be specified for the container that contains the toggletip content to ensure screen reader focus is received and the information can be acquired.
+-   Toggletip content **SHOULD** remain persistent unless dismissed.
+
+### Mouse Interactions
+
+-   The toggle-tip popover should appear when a user clicks the trigger button, and disappear when the user clicks anything other than the popover.
+-   Scrolling or zooming the view should not close the popover.
+
+### Non-Mouse Interactions
+
+| Key                | Description                                                    |
+| ------------------ | -------------------------------------------------------------- |
+| `TAB`              | Moves focus to next focusable element                          |
+| `SPACE` or `ENTER` | Opens/closes toggle-tip popover when trigger button is focused |
+
+`TAB` to the toggle-tip trigger button moves focus to the trigger button with a focus ring (i.e. the key focus state).
+
+`ENTER` or `SPACE` on a focused trigger button opens the popover. Key focus remains on the trigger button, but the trigger button changes to the select state.
+
+![Toggle tip keyboard open](./spec-images/key-toggle-open.png)
+
+Pressing `ENTER` or `SPACE` a second time (or `ESC`) closes the popover.
+
+![Toggle tip keyboard close](./spec-images/key-toggle-close.png)
+
+Pressing `TAB` while the popover is open moves focus to the first focusable element in the content area.
+
+![Toggle tip keyboard tab into popover](./spec-images/key-tab-popover.png)
+
+Pressing `TAB` while focused on the last focusable element in the content area will move focus to the next focusable element on the main page. Moving the focus **SHOULD NOT** automatically close the popover. However, interacting with another focusable element outside of the popover **MUST** close the popover.
```

Comment:
It might be worth noting that in the two design systems I looked at, [Carbon](https://carbondesignsystem.com/components/toggletip/usage/) dismisses the popover after you tab passed the last element, and [Workbench](https://workbench.gusto.com/components/toggletip/) cycles between the elements in the popover.

Reply:
Great points - Looking more closely at the Carbon example, it's clear and non-destructive to simply close the popover on focus change, so I like that approach. I've made the change.

I don't like Workbench's approach as much as they've made something more modal than we want, and the focus trapping behavior (and close button) supports that.